### PR TITLE
[JFG-964] Tests generated from archetype for java builder doesn't work

### PR DESCRIPTION
**Pull request by a-badaev**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java [hunk 1]
```diff
@@ -37,25 +39,49 @@
...
+
+        Configuration configuration = configurations.get(jTestSuiteNameToExecute);
+        if (configuration == null) {
+            throw new IllegalArgumentException(String.format("No Jagger configuration with name %s", jTestSuiteNameToExecute));
+        }
+        return configuration;
+    }
+    
+    
     public Configuration generate(String userJTestSuiteName) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java]: this one looks useless. propose to put all the code to the generate() method

Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java]: it can come in handy in tests

Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java]: i mean if you get rid of it there are no way to trigger any other test except the one with the name you passed to the constructor

Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java]: don't see problem with it
- if you are in stand by mode - there is a way to do it via jaas
- if not => one run = one configuration from constructor

**Pull request merged**